scripts/gitlab/lingua.dic

```diff
@@ -28,6 +29,8 @@ broadcast/UDSMG
 BTC/S
 canonicalize/D
 CentOS
+Chain1
+Chain2
```

Comment:
is this still needed? you only touch toml files here, I don't see how this might trigger spellcheck cc @drahnr

Reply:
There are a few things to note: The spellcheck run is done on files that changed in respect to the changes git knows. This builds on-top of changes files between the base commit of a branch and the branch's head commit. Now it seems occasionally (for whatever reason) gitlab picks up more files than anticipated `${CI_COMMIT_SHA} $(git merge-base ${CI_COMMIT_SHA} ${CI_DEFAULT_BRANCH})` so this might be necessary to make that check green.

Besides that, I don't  think we should add those on-off items.  It should be marked as code I guess, like you do on github `ThisChain`, `Chain1`, and `Chain2`.

Reply:
Isn't it just going to nag someone else later if I don't include them? Is there a special way to mark a word as not a word but to be ignored if found?

Reply:
I think @drahnr suggests to put such words in backticks (`) instead of adding to the dictionary

Reply:
ok - as these are in the bridge subtree I will revert the dict changes here and will add backticks when I remove the unneeded dependencies from the bridges repo.